tools/RepoTasks/RepoTasks.Cmdlets.Tests/NewFormatPs1XmlCommandShould.cs

```diff
@@ -143,13 +141,14 @@ public void SaveOnlyMarkedPropertiesInSpecifiedViews()
                             // entrytype (1), prop-name or script-block (2), target (3), label (4), table-column-width (5), position (6), group-by-this-param (7)
                             new List<Tuple<EntryType, string, View, string, uint?, uint?, bool>>
                             {
-                                Tuple.Create<EntryType, string, View, string, uint?, uint?, bool>(EntryType.PropertyName, "Id", View.List, null, 0, null, false),
-                                Tuple.Create<EntryType, string, View, string, uint?, uint?, bool>(EntryType.PropertyName, "RequestId", View.Both, null, null, 1, false),
+                                Tuple.Create<EntryType, string, View, string, uint?, uint?, bool>(EntryType.ScriptBlock, "S_.Foo", View.List, "PsDummyOutput1 Id", 0, null, false),
```

Comment:
@vladimir-shcherbakov should `S_.Foo` be `$_.Foo` for consistency with other script blocks in the tests?